ArduCopter/GCS_Mavlink.cpp

```diff
@@ -624,6 +624,14 @@ void GCS_MAVLINK_Copter::handle_landing_target(const mavlink_landing_target_t &p
 #endif
 }
 
+bool GCS_MAVLINK_Copter::set_destination_target(const Location &dest_loc, bool use_yaw, float yaw_cd, bool use_yaw_rate, float yaw_rate_cds) const {
+    bool result = false;
+#if MODE_GUIDED_ENABLED == ENABLED
+    result = copter.mode_guided.set_destination(request_location, use_yaw, yaw_cd, use_yaw_rate, yaw_rate_cds);
```

Comment:
Variable `request_location` here is a mistake. It should be `dest_loc`

Reply:
This was fixed